infrastructure/src/main/java/org/corfudb/infrastructure/BatchWriter.java

```diff
@@ -107,13 +107,14 @@ public void bulkWrite(List<LogData> entries, long epoch) {
     /**
      * Trim addresses from log up to a prefix.
      *
-     * @param address prefix address to trim to (inclusive)
+     * @param address     prefix address to trim to (inclusive)
+     * @param prefixEpoch epoch stamped on the trim request.
      */
-    public void prefixTrim(@Nonnull Token address) {
+    public void prefixTrim(@Nonnull Token address, long prefixEpoch) {
         try {
```

Comment:
Redundant information. Just pass a token with an un-initialized epoch.

Reply:
I still need the epoch for the BatchWriter epoch validation. I will change it to carry the prefixEpoch in the token's epoch.